### PR TITLE
Have the model metadata as part of the reponse

### PR DESCRIPTION
**Pull request by kohn1001**
**Proposed changes**:
- We need the model metadata as part of the response - more specifically the model name (trained time), very useful for debugging - making sure you are working w/ a specific trained model

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [x] updated the changelog
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa_nlu#code-style) for instructions)

### CODE REVIEW

**Comment by erohmensing** [comment]
Hi @kohn1001, thanks for the PR! Looks like your addition caused a test to break -- I think it's just a case of needing to handle the case where the `model_metadata` doesn't exist. In addition, changing the output of the `/parse` endpoint should also include an update to the API spec here: https://github.com/RasaHQ/rasa/tree/master/docs/_static/spec

**Comment by tabergma** [comment]
The current change will update the output of the model when processing a message. Instead of 
```
{"intent": {"name": None, "confidence": 0.0}, "entities": []}
```
it would then return something like 
```
{"intent": {"name": None, "confidence": 0.0}, "entities": [], "metadata": "model": "<some-time>"}
```
I'm not sure if we want that as this information is static. Can't read this information from the model itself when loading it? Why do you need it when using the model to predict something?

**Comment by tabergma** [comment]
@kohn1001 We discussed it internally and it seems to be a good idea to add some metadata to the response. Will review the PR later today.

File: rasa/nlu/model.py [hunk 1]
```diff
@@ -381,4 +381,10 @@ def parse(
 
         output = self.default_output_attributes()
         output.update(message.as_dict(only_output_properties=only_output_properties))
+
+        if self.model_metadata:
+            output.update(
+                {"metadata": {"model": self.model_metadata.get("trained_at")}}
```
Comment (tabergma) [inline rasa/nlu/model.py]: The names are not very clear in my opinion. `model` does not seem to be the same thing as `trained_at`.

**Comment by erohmensing** [comment]
@kohn1001 are you still interested in implementing this?

File: rasa/nlu/model.py (hunk 1 above)
Comment (kohn1001) [reply rasa/nlu/model.py]: Yes - but you build the model name from the timestamp that it was trained on.

**Comment by kohn1001** [comment]
yes - but i need your assistance w/ the remaining 3 checkboxes

**Comment by tabergma** [comment]
@kohn1001 Great! 

First of all, please merge the current master into your branch as your branch is out-of-date. After you updated the branch, you will see that we change the way how we report changelog entries. Please take a look here: https://github.com/RasaHQ/rasa/tree/master/changelog.

Regarding the test: Take a look at https://github.com/RasaHQ/rasa/blob/master/tests/nlu/training/test_train.py. You can add a test here that checks that metadata are present in the response.

**Comment by tmbo** [comment]
I'll close this for now. Let me know if you still want to continue working on this.

**Pull request closed without merging**